src/Symfony/Bundle/TwigBundle/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+master
```

Comment:
should be 4.3.0

Reply:
Ok, I've add a test case for this feature and updated the changelog to refer the version 4.3.0